### PR TITLE
67 add media

### LINKED ISSUE #52: Sequence Diagram Mini-Lab adding images to fit the examples
For me it's hard to imagine some of the plantUML code, so I think adding images to the steps could be helpful. Giving an example of an alt output could be helpful to add at the end.

### PR DESCRIPTION
**Pull request by nfantinodyer**
fixes #52

### CODE REVIEW

**Review by onzfonz — CHANGES_REQUESTED** [review]
please rename the files something closer to what they represent, something like basic_plantuml_diagram or something along those lines

Please also change the text description inside of the brackets for the pictures

**Comment by nfantinodyer** [comment]
Changes have been addressed

File: docs/labs/67-Sequence.md [hunk 1]
```diff
@@ -113,6 +117,8 @@ that could arise in the interaction,
 but I tend not to use them too much
 because then we get into this idea of having lots of logic in our sequence diagrams,
 which then makes it more difficult to understand.
+But this is the potential if we use ```alt``` and ```else``` in our sequence diagrams:
+![advanced_plantuml_diagram](lab67media/advanced_plantuml_diagram.png)
```
Comment (onzfonz) [inline docs/labs/67-Sequence.md:121]:
```suggestion
![advanced sequence diagram with alt and else](lab67media/advanced_plantuml_diagram.png)
```

File: docs/labs/67-Sequence.md [hunk 2]
```diff
@@ -93,6 +93,10 @@ like so.
 Obj2-->Obj1 : <i>[returnedObject]</i>
 ```
 
+Here is an example diagram:
+
+![basic_plantuml_diagram](lab67media/basic_plantuml_diagram.png)
```
Comment (onzfonz) [inline docs/labs/67-Sequence.md:98]:
```suggestion
![basic sequence diagram](lab67media/basic_plantuml_diagram.png)
```

**Review by onzfonz — CHANGES_REQUESTED** [review]
Just approve these two from the pull request conversation page and you should be good

**Comment by onzfonz** [comment]
@nfantinodyer can you approve these two requests so that this can be merged?

**Comment by nfantinodyer** [comment]
@onzfonz It says I'm unable to approve the changes as its my pull request, so I clicked on the resolve conversation button for those two changes instead.
image

**Comment by onzfonz** [comment]
In the conversation it should show the suggestions and say commit suggestion

**Comment by nfantinodyer** [comment]
@onzfonz I found it thank you

**Pull request closed without merging**